packages/renderer/src/lib/table/Table.svelte

```diff
@@ -66,6 +67,16 @@ function sort(column: Column<any>) {
   data = data.sort(comparator);
 }
 
+onMount(async () => {
+  if (defaultSortColumn) {
+    const column: Column<any> | undefined = columns.find(column => column.title === defaultSortColumn);
```

Comment:
I'm wondering if we need the if (defaultSortColumn) as if it's not defined it won't match as it looks like that title is not optional

Reply:
(also I'm still approving, my comments were not "required for changes" request)

Reply:
Understood. I figured I may as well make the changes while I waited for another approval. I've removed the if, and if there are no further comments in the next hour I think it has been long enough/is small enough that I'll merge.